templates/problem/voting-controls.html

```diff
@@ -0,0 +1,206 @@
+<style>
+    .vote-form {
+        background-color: rgb(255,255,255);
+        padding: 40px;
+        border-radius: 25px;
+        justify-content: center;
+        align-items: center;
+        margin: auto;
+        text-align: center;
+    }
+
+    .vote-form-info {
+        font-size: 24px;
+        font-weight: bold;
+        padding:10px
+    }
+
+    .vote-form-value {
+        font-size: 24px;
+    }
+
+    .vote-form-text {
+        font-size: 18px;
+    }
+
+    .voting-form-error {
+        font-size: 18px;
+        color: red;
+    }
+
+    .vote-button {
+        float:right;
+    }
+
+    .form-button {
+        cursor: pointer;
+    }
+</style>
+
+{% if not request.in_contest %}
+    <hr style="padding-top: 0.1em">
+{% endif %}
+
+{% include 'lightbox.html' %}
+
+{% if can_vote %}
+    <div class="vote-form" id="id_vote_form_box">
+        <div id="id_has_voted_prompt">
+            <span class="vote-form-info">{{ _('Current Vote:') }}</span>
+            <span id="id_current_vote_value" class="vote-form-value"></span>
+            <br/>
+            <br/>
+            <span class="vote-form-text">{{  _('Change your vote below!') }}</span>
+            <br/>
+            <br/>
+        </div>
+        <div id="id_has_not_voted_prompt">
+            <span class="vote-form-text">{{ _("Cast your vote on this problem's weightage below!") }}</span>
```

Comment:
I was going to let this through but the more I look at it, the weirder this line feels. Can you find a better word than `weightage`?

Reply:
points?